docs/fidesops/docs/development/documentation.md

```diff
@@ -14,7 +12,7 @@ To write concept docs, add Markdown files to the `docs/fidesops/docs/` directory
 
 ### Capitalization
 
-Concepts that refer to proper nouns or are trademarked should always be capitalized, including "Fides" and "Fidesops".
+Concepts that refer to proper nouns or are trademarked should always be capitalized, including "Fides".
```

Comment:
This could be misleading — should we explicitly state not to capitalise fidesops?

Reply:
I want to do a more comprehensive rewrite of this to encompass the new tone/style changes we're still finishing up - I added the note about using lowercase for Fides tools, but will include more docs on headers/titles when the guide is finished up.